GTG/gtk/preferences.py

```diff
@@ -26,10 +26,8 @@
 from gi.repository.Gdk import Color
```

Comment:
This can be joined with the line above to:

from gi.repository import Gtk, Gdk